docs/config.md

```diff
@@ -1456,13 +1456,17 @@ services. It is an error for both of these files to exist.
 
 The location of the `jj` config file can also be overridden with the
 `JJ_CONFIG` environment variable. If it is not empty, it should contain the path
-to a TOML file that will be used instead of any configuration file in the
-default locations. For example,
+to a TOML file or a directory containing TOML files that will be used instead
+of any configuration file in the default locations. For example,
```

Comment:
How about:

If it is not empty, it will be used instead of any configuration file in the default locations.  If it is a path to a TOML file, then that file will be loaded instead.  If it is a path to a directory, then all the TOML files in that directory will be loaded in lexicographic order and merged.  This allows you to split your configuration across multiple files and can combine well with [Conditional Variables](#Conditional Variables).

Reply:
Thanks, I also felt that the wording became a bit awkward. I still kept the existing example, let me know if it would be useful to extend or remove that.